libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview.tsx

```diff
@@ -8,21 +8,35 @@ import {
   OverallMetricChart
 } from "@responsible-ai/core-ui";
 import { localization } from "@responsible-ai/localization";
-import _ from "lodash";
-import { Text } from "office-ui-fabric-react";
+import { Stack, Text } from "office-ui-fabric-react";
 import React from "react";
 
-import { modelOverviewStyles } from "./ModelOverview.styles";
+interface IModelOverviewProps {
+  showNewModelOverviewExperience: boolean;
+}
 
-class ModelOverviewProps {}
+// This will be an interface shortly.
+// As long as it has no members it needs to remain a class for code style reasons.
+class IModelOverviewState {}
 
-export class ModelOverview extends React.PureComponent<ModelOverviewProps> {
+export class ModelOverview extends React.Component<
+  IModelOverviewProps,
+  IModelOverviewState
+> {
   public static contextType = ModelAssessmentContext;
   public context: React.ContextType<typeof ModelAssessmentContext> =
     defaultModelAssessmentContext;
 
+  constructor(props: IModelOverviewProps) {
+    super(props);
+    this.state = {
+      selectedMetrics: [],
```

Comment:
I see some linting warnings about this not being used

Reply:
![image](https://user-images.githubusercontent.com/24683184/161340486-6d157b75-e0b9-4fc0-aeee-286c00fb836e.png)

Reply:
I suppose I didn't delete all the unused parts from my larger feature branch. Well spotted! Will remove...